gems/aws-sdk-core/lib/aws-sdk-core/profile/profile_file_standardizer.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+module AWS::SDK::Core
+  # @api private
+  class ProfileFileStandardizer
+    def initialize(raw_file, file_type)
+      @raw_file = raw_file
+      @file_type = file_type
+    end
+
+    def standardize
```

Comment:
I know this is API private, but some comments/docs would help with readability.  I had a hard time understanding what this is supposed to be doing.

Reply:
I'll add docs when I can